app/services/collators/housing_costs_collator.rb

```diff
@@ -11,8 +11,8 @@ def self.blank
 
     class << self
       def call(housing_cost_outgoings:, gross_income_summary:, submission_date:, person:, allow_negative_net:)
-        housing_calculator = Calculators::HousingCostsCalculator.new(housing_cost_outgoings:, gross_income_summary:,
-                                                                     submission_date:, person:)
+        housing_calculator = Calculators::HousingCostsCalculator.call(housing_cost_outgoings:, gross_income_summary:,
+                                                                      submission_date:, housing_cost_cap: housing_costs_cap_apply?(person))
```

Comment:
@starswan Should we change attribute name from `housing_cost_cap`  to `housing_cost_cap_applies` to reflect boolean value?